tests/unit/restapi/v1/test_workflows.py

```diff
@@ -0,0 +1,234 @@
+            "outputParams": [
+                {
+                    "name": "greeting",
+                    "parameterType": 2,
+                },
+            ],
+        },
+    ]
+    plugin_file_response = actions.register_plugin_file(
+        client,
+        plugin_id=plugin_response["id"],
+        filename="tasks.py",
+        description="The task plugin file for hello world.",
+        contents=plugin_file_contents,
+        tasks = plugin_file_tasks,
+    ).get_json()
+    task_graph = textwrap.dedent(
+        """# my entrypoint graph
+        hello_step:
+          hello_world:
+            name: $name
+        """
+    )
+
+    plugin_ids = [plugin_response["id"]]
+    entrypoint_parameters = [
+        {
+            "name" : "name",
+            "defaultValue": "User",
+            "parameterType": "string",
+        },
+    ]
+    assert_entrypoint_workflow_is_valid(
+        client,
+        task_graph=task_graph,
+        plugin_ids=plugin_ids,
+        entrypoint_parameters=entrypoint_parameters,
+    )
+
+
+def test_entrypoint_workflow_validation_has_error(
+    client: FlaskClient,
+    db: SQLAlchemy,
+    auth_account: dict[str, Any],
+) -> None:
+    """"""
+    plugin_response = actions.register_plugin(
+        client,
+        name="hello_world",
+        description="The hello world plugin.",
+        group_id=auth_account["default_group_id"],
+    ).get_json()
+    plugin_file_contents = textwrap.dedent(
+        """"from dioptra import pyplugs
+        
+        @pyplugs.register
+        def hello_world(name: str) -> str:
+            return f'Hello, {name}!'"
+        """
+    )
+    plugin_file_tasks = [
+        {
+            "name": "hello_world",
+            "inputParams": [
+                {
+                    "name": "name",
+                    "parameterType": 2,
+                    "required": True,
+                },
+            ],
+            "outputParams": [
+                {
+                    "name": "greeting",
+                    "parameterType": 2,
+                },
+            ],
+        },
+    ]
+    plugin_file_response = actions.register_plugin_file(
+        client,
+        plugin_id=plugin_response["id"],
+        filename="tasks.py",
+        description="The task plugin file for hello world.",
+        contents=plugin_file_contents,
+        tasks = plugin_file_tasks,
+    ).get_json()
+    task_graph = textwrap.dedent(
+        """# my entrypoint graph
+        hello_step:
+          hello_wrld:
+            name: $name
+        """
+    ) # task graph is wrong, hello_wrld is not the task plugin
+
+    plugin_ids = [plugin_response["id"]]
+    entrypoint_parameters = [
+        {
+            "name" : "name",
+            "defaultValue": "User",
+            "parameterType": "string",
+        },
+    ]
+    expected_message = "[ValidationIssue(IssueType.SEMANTIC, IssueSeverity.ERROR, 'In step \"hello_step\": unrecognized task plugin: hello_wrld')]"
```

Comment:
I don't think we should match the error text exactly. Maybe just check response code and the issue type.